pyvista/examples/examples.py

```diff
@@ -412,3 +412,95 @@ def load_nut():
 
     """
     return pyvista.read(os.path.join(dir_path, 'nut.ply'))
+
+
+def load_hydrogen_orbital(n=1, l=0, m=0, zoom_fac=1.0):
+    """Load the hydrogen wave function for a :class:`pyvista.UniformGrid`.
+
+    This is the Schrödinger equation evaluated in three dimensions.
```

Comment:
```suggestion
    This is the solution to the Schrödinger equation for hydrogen
    evaluated in three-dimensional Cartesian space.
```
The "Cartesian space" part is not strictly necessarym but the solution of the equation involves polar coordinates so it might make sense to emphasize that the final result here is in "real space".